### PR TITLE
fix: make jobs in Draft.upload work as max threads when upload files

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

File: graviti/manager/draft.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def commit(self, title: str, description: Optional[str] = None) -> Commit:
 
         return Commit(self._dataset, **commit_info)
 
-    def upload(self, jobs: int = 1) -> None:  # pylint: disable=unused-argument
+    def upload(self, jobs: int = 8) -> None:  # pylint: disable=unused-argument
```
Comment (linjiX) [inline graviti/manager/draft.py:192]: shall we mention the default multi-thread behavior in the docstring?

Comment (graczhual) [inline graviti/manager/draft.py:192]: do not need this pylint

Comment (linjiX) [reply graviti/manager/draft.py:192]: Shall we remove the pylint comment?

File: graviti/openapi/file.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def upload_files(
     draft_number: int,
     sheet: str,
     files: Iterable[File],
-    jobs: int = 1,
+    jobs: int = 8,
```
Comment (linjiX) [inline graviti/openapi/file.py:85]: can we remove the default value here?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/operation/frame.py [hunk 3]
```diff
@@ -28,6 +28,7 @@ def do(  # pylint: disable=invalid-name
         *,
         draft_number: int,
         sheet: str,
+        jobs: int = 8,
```
Comment (linjiX) [inline graviti/operation/frame.py:31]: shall we also remove the default value here?

File: graviti/utility/requests.py [hunk 4]
```diff
@@ -226,14 +226,14 @@ def read(self, amt: Optional[int] = None) -> bytes:
 
 
 def submit_multithread_tasks(
-    function: Callable[[_T], Any], arguments: Iterable[_T], *, jobs: int = 1
+    function: Callable[[_T], Any], arguments: Iterable[_T], *, jobs: int = 8
```
Comment (linjiX) [inline graviti/utility/requests.py:229]: and here?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

**Review by graczhual — APPROVED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**